spec/controllers/api/v1/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    let!(:book_suggestion) { create(:book_suggestion) }
+    context 'When creating a valid suggestion' do
+      before do
+        post :create, params: { book_suggestion: {
```

Comment:
Remember you have `attributes_for :book_suggestion`, which uses the factories instead of hardcoding the hashes